bin/check-process.rb

```diff
@@ -261,7 +270,10 @@ def run
     procs.reject! { |p| p[:command] =~ /#{config[:exclude_pat]}/ } if config[:exclude_pat]
     procs.reject! { |p| p[:command] !~ /#{config[:cmd_pat]}/ } if config[:cmd_pat]
     procs.select! { |p| p[:vsz].to_f > config[:vsz] } if config[:vsz]
+    # Ensure RSS is over this value
     procs.select! { |p| p[:rss].to_f > config[:rss] } if config[:rss]
+    # Ensure RSS is under this value
```

Comment:
Perhaps I'm misunderstanding the purpose of or proper way to use this plugin. I simply want to add the functionality to enable me to take a check that currently verifies that the process is running (which this plugin already accomplishes) and additionally alert if the RSS for this process goes over some critical threshold.

It made sense to me to take my current check: `check-process.rb -p /opt/lita/bot -u lita` which, when this process is running, returns `OK`, and add some parameter to tell the check to trigger an alert (meaning not return `OK`) if the RSS value goes over some value.

Many of the options for this plugin seem unintuitive looking over the actual code and option descriptions, because, for example `vsz` has a description of `Trigger on a Virtual Memory size is bigger than this` but yet the actual comparison _selects_ processes only if their value is over that value: `procs.select! { |p| p[:vsz].to_f > config[:vsz] }`. So if you specify `-z 999` and the process has a VSZ of 1000, it seems like the plugin will return `OK`.

So I am unclear if the intention with CLI options is to declare under what conditions to alert (thus `over` would be more appropriate), or to declare conditions a process must match (which would make `under` more appropriate). Given that most descriptions read as "Trigger... if bigger..." but the actual reducing of the process list seems to do the opposite, I could use guidance.